ISS-175257: Fix outdated marketplace docs (#230)
devrev/fern-api-docs: fern/docs/pages/snapin-concepts.mdx

The [marketplace](https://marketplace.devrev.ai/marketplace) is a portal where DevRev and external developers can publish snap-ins and other extensions.

> In case the developers want to make their snap-ins available to only their dev org or some external ones, they can publish them to marketplace with `restricted` access.

Author: I've had it written better and ek-line corrected it for me 😅

> In case the developers want to make their snap-ins available to only their dev org and optionally to some external ones, they can publish them to marketplace with `restricted` access.

Reviewer: ```suggestion
If developers want to make their snap-ins available only to their dev org, and optionally to some external ones, they can publish them to the marketplace with `restricted` access.
```
